.gitlab-ci.yml

```diff
@@ -44,3 +44,13 @@ run tests:
   only:
     - branches
     - merge_requests
+
+run kcov:
+  stage: test
+  image: docker:latest
+  services:
+    - docker:dind
+  script:
+    - docker --version
```

Comment:
do we need this?

Reply:
Well, it‘s additional debug info. Just a habit to print version numbers ^^